src/ansys/sound/core/sound_power/sound_power_level_iso_3744.py

```diff
@@ -0,0 +1,616 @@
+# Copyright (C) 2023 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""Computes ISO 3744 sound power level."""
+import warnings
+
+from ansys.dpf.core import Field, FieldsContainer, Operator
+import matplotlib.pyplot as plt
+import numpy as np
+from numpy import typing as npt
+
+from . import SoundPowerParent
+from .._pyansys_sound import PyAnsysSoundException, PyAnsysSoundWarning
+
+ID_COMPUTE_SOUND_POWER_LEVEL = "compute_sound_power_level_iso3744"
+ID_LOAD_SOUND_POWER_LEVEL = "load_project_sound_power_level_iso3744"
+LW = 0
+LWA = 1
+LW_OCTAVE = 2
+LW_THIRDOCTAVE = 3
+LW_OCTAVE_NP = 2
+OCTAVE_CENTER_FREQUENCIES = 3
+LW_THIRDOCTAVE_NP = 4
+THIRDOCTAVE_CENTER_FREQUENCIES = 5
```

Comment:
@ansaminard I would add comments to this id you defined here (it is hard to understand what they are made for, and most of all why some of them have the same value!)
We can also maybe rename them by adding "_ID" or "INDEX" at the end, or at the beginning as you made for the operators' just names above.